.gitignore

```diff
@@ -19,6 +19,9 @@ package-lock.json
 # dependencies
 node_modules/
 
+#AWS JSON file
+/src/app/config/cla-env-config.json
```

Comment:
we should rename this file to something like: `/src/app/config/vd-env-config.json`